content/monitors/monitor_uptime_widget.md

```diff
@@ -49,8 +51,9 @@ If the chosen monitor for the widget is configured to alert on *WARN* or *NO DAT
 
```

Comment:
If the chosen monitor for the widget is configured to alert on *NO DATA*, it's rendered as down (red) inside the uptime widget.

Reply:
“warn“ doesn't count as a down state ?